MaxText/input_pipeline.py

```diff
@@ -216,11 +217,12 @@ def preprocess_dataset(config: ml_collections.ConfigDict,
       tokenizer.TokenizeOp(sp_tokenizer), num_parallel_calls=AUTOTUNE)
 
   # Set global batch size.
-  batch_size = config.per_device_batch_size * global_mesh.size
+  global_batch_size_to_load = config.global_batch_size_to_load
```

Comment:
I didn't see global_batch_size_to_load in yml, what's up?

Reply:
global_batch_size_to_load is defined in [pyconfig.py](https://github.com/google/maxtext/blob/main/MaxText/pyconfig.py#L114)